traitsui/qt4/tabular_editor.py

```diff
@@ -507,7 +507,7 @@ def _selected_column_changed(self, selected_column):
                 # Once selected, scroll to the column
                 self.scroll_to_column = selected_column
 
-    scroll_to_row_hint_map = {
+    scroll_to_position_hint_map = {
```

Comment:
tangential comment - not sure why this is a class attribute instead of being a module-level constant